modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/service/test/SegmentsExperimentLocalServiceTest.java

```diff
@@ -679,12 +679,16 @@ public void testUpdateSegmentsExperimentStatusToCompletedWithWinnerSegmentsExper
 
 		SegmentsExperience segmentsExperience1 = _addSegmentsExperience();
 
+		int prioritySegmentsExperience1 = segmentsExperience1.getPriority();
```

Comment:
Better if you add a test case per use case, for example, if the variant is in 5 and the control in -2 ...

Reply:
I like, one for a positive control and another for a negative one

Reply:
No, please, one per priority possibility

Edges (0, -2) positive and negative.

Reply:
;)

Reply:
Added test for the following use cases:

- control variant with the default experience so the default priority, which is -1
- control variant with priority 0
- control variant with priority -2